### ISSUE TITLE
Deserialization: not correctly utilizing the load key

### ISSUE THREAD

**Issue by JoeHolt** [ericb/ciri#5]
In this example:

```python
class DocumentOverlayElementSchema(PolySchema):
    element_id = f.String(load='id', name='id', required=True)
    label = f.String(name='label', allow_none=True)
    field_type = f.String(load='type', required=True) 
    __poly_on__ = field_type
```
I am reading a JSON that looks like the following and is built dynamically via the poly:
```python
{
  "element_id": "uuid",
  "x_position": 74.3515625,
  "y_position": 48.6875,
  "z_position": 0,
  "width": 94.5,
  "height": 14.5,
  "output_type": "text",
  "value": "Hello world",
  "type": "textarea",
  "label": "Field Name",
  "field_definitions": {},
  "styling": {
    "font_size": null,
    "font_bold": false,
    "font_uppercase": false,
    "text_alignment": "left",
    "text_wraps": true,
    "line_height": null,
    "first_line_indent": false
  },
  "conditional_logic": {
    "conditional_on": "C56A4180-65AA-42EC-A945-5FD21DEC0538",
    "conditional_value": "0000"
  },
  "custom_output_string": null,
  "business_logic": null
}
```
Here, I want to read the element with the key `type` into a python field called `field_type`, which I then use to poly on via `__poly_on__ = field_type. The problem is, despite specifying a the JSON key via `load='type`, the poly can not recognize this and throws a serialization error.

I suspect this issue lies in the `load='type'` not referencing the json object correctly, not poly directly. Let me know if more clarification is necessary.

**Comment by pegex-dev** [comment]
fixed in 175ed9dff9ac1fdd159a1b373065bd4b682bde82
This issue made me realize that the load key should only affect deserialization.

**Comment by ericb** [comment]
fixed with 175ed9dff9ac1fdd159a1b373065bd4b682bde82

This issue made me realize that the `load` key should only affect during deserialization.
